src/write_gcm.c

```diff
@@ -4162,6 +4169,12 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
       queue->tail = NULL;
     }
     --queue->size;
+    ++queue->drop_count;
+    if ((queue->drop_count % QUEUE_DROP_REPORT_LIMIT) == 0) {
+      WARNING("write_gcm: %s queue dropped %d metric points due to dispatch"
+              " backlog.",
+              queue_name, QUEUE_DROP_REPORT_LIMIT);
```

Comment:
Done. There was an example above with variables on a separate line in case of a break but I guess there all the arguments did fit on one line. Updated.